Python/Product/PythonTools/PythonTools/OutliningTaggerProvider.cs

```diff
@@ -199,6 +199,14 @@ internal class OutliningTagger : ITagger<IOutliningRegionTag> {
                 int cnt = 0;
                 var text = snapshot.GetText(start, length);
 
+                // The VS Provider is fragile to us splitting in between \r\n.  
+                // Check here that we haven't done that and correct the error.
+                if (text[0] == '\n' && start >= 1 && snapshot.GetText(start - 1, 1) == "\r") {

```

Comment:
So the issue is that I advance by one extra to account for ':' in class so it was visible.

1) do this
2) fix an issue with class defs where the ':' is not included on the first line.

This is kind of a patchwork job though so I will fix the root issue. I will remove our offset and fix the class issue.  Update coming.